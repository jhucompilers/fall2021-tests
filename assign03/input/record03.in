PROGRAM record03;
  TYPE int = INTEGER;
  VAR x : RECORD
    int : int;
  END;
BEGIN
    READ x.int;
END.