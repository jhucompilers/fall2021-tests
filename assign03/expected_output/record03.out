0,TYPE,int,INTEGER
1,VAR,int,INTEGER
0,VAR,x,RECORD (INTEGER)
